src/include/sof/common.h

```diff
@@ -29,6 +29,8 @@
 
 #define __aligned(x) __attribute__((__aligned__(x)))
 
+#define ffs(i) __builtin_ffs(i)
```

Comment:
@tlauda @lyakh  Maybe that's not the best place for this macro, just because it's __builtin. What if used compiler doesn't support it? It's more like a math function, that may be replaced with builtin if compiler allows.

Reply:
@jajanusz Maybe, do you have a better idea? Do you have any specific compilers which don't implement this which we use? I'd propose to keep this until we find a better solution. FWIW it was a suggestion from Liam to put it originally in sof.h, now these macros got moved to common.h

Reply:
@lyakh It was a suggestion, so you would rethink it. It's more about what domain macro/function belong than what's supported. For now we don't support any compiler that doesn't have that feature and I don't know of any plans of doing it, so it may stay in common.h for now. Feel free to rebase so we can merge this one.